WcaOnRails/app/views/devise/passwords/edit.html.erb

```diff
@@ -6,9 +6,8 @@
     <div class="panel-body">
       <%= simple_form_for(resource, :as => resource_name, :url => password_path(resource_name), :html => { :method => :put, role: "form" }) do |f| %>
         <%= f.hidden_field :reset_password_token %>
-
-        <%= f.input :password, autofocus: true  %>
-        <%= f.input :password_confirmation  %>
+        <%= f.input :password, label: t('.new_password'), autofocus: true  %>
```

Comment:
I'm surprised to not see this added to `en.yml`. Is this already defined somewhere?

Reply:
This is part of devise translation from gem. https://github.com/tigrish/devise-i18n.  So it will automatically also translate to other languages.

Reply:
https://github.com/tigrish/devise-i18n/blob/master/rails/locales/en.yml#L82